firestore/query.go

```diff
@@ -1226,14 +1459,17 @@ type QuerySnapshot struct {
 
 type btreeDocumentIterator btree.Iterator
 
-func (it *btreeDocumentIterator) next() (*DocumentSnapshot, error) {
+func (it *btreeDocumentIterator) next(_ ...RunOption) (*DocumentSnapshot, error) {
```

Comment:
Is there a reason why a `RunOption` would need to be passed in here despite the function not needing it? Is it to futureproof this call?

Reply:
```
type docIterator interface {
	next(opts ...RunOption) (*DocumentSnapshot, error)
	getExplainMetrics() *ExplainMetrics
	stop()
}
```

docIterator interface has been modified.  RunOption is being passed here to ensure that the btreeDocumentIterator implements the docIterator interface.

Yes, it is not getting used currently but might get used later